diff --git a/ldastoolsal/test/CMakeLists.txt b/ldastoolsal/test/CMakeLists.txt
index e7e3d56ee..323852d1a 100644
--- a/ldastoolsal/test/CMakeLists.txt
+++ b/ldastoolsal/test/CMakeLists.txt
@@ -192,7 +192,7 @@ cx_target_test( tGPSTime COMMAND tGPSTime )
 #------------------------------------------------------------------------
 
 cx_msg_debug_variable( CMAKE_SYSTEM_PROCESSOR )
-if ( NOT APPLE AND NOT CMAKE_SYSTEM_MACHINE STREQUAL aarch64 )
+if ( NOT APPLE AND NOT CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64 AND NOT CMAKE_SYSTEM_PROCESSOR STREQUAL ppc64le )
   set( tPSInfo_SOURCES tPSInfo.cc )
   cx_target_executable(
     tPSInfo
