diff --git a/ldastoolsal/ldastoolsal-uninstalled.pc.in b/ldastoolsal/ldastoolsal-uninstalled.pc.in
index 6cdc1b138..3b9a8c973 100644
--- a/ldastoolsal/ldastoolsal-uninstalled.pc.in
+++ b/ldastoolsal/ldastoolsal-uninstalled.pc.in
@@ -11,5 +11,6 @@ SWIGFLAGS=-I${abs_top_builddir}/include
 Name: ldastoolsal
 Description: Library which implements abstraction layer for the ldas-tools package.
 Version: @PACKAGE_VERSION@
-Libs: ${abs_top_builddir}/src/libldastoolsal.la @BOOST_LDFLAGS@ ${CXXSTDLDFLAGS}
+Libs: ${abs_top_builddir}/src/libldastoolsal.la
+Libs.private: ${abs_top_builddir}/src/libldastoolsal.la @BOOST_LDFLAGS@ ${CXXSTDLDFLAGS}
 Cflags: ${CXXSTDFLAGS} -I${abs_top_builddir}/include @BOOST_CPPFLAGS@
diff --git a/ldastoolsal/src/ldastoolsal.pc.in b/ldastoolsal/src/ldastoolsal.pc.in
index 6ece6fab5..a2214183e 100644
--- a/ldastoolsal/src/ldastoolsal.pc.in
+++ b/ldastoolsal/src/ldastoolsal.pc.in
@@ -10,5 +10,6 @@ CXXSTDLDFLAGS=@CXXSTDLDFLAGS@
 Name: ldastoolsal
 Description: Library which implements abstraction layer for the ldas-tools package.
 Version: @PROJECT_VERSION@
-Libs: -L${libdir}  -lldastoolsal ${CXXSTDLDFLAGS} @BOOST_LDFLAGS@ @STDCXXFS_LIBS@
+Libs: -L${libdir}  -lldastoolsal
+Libs.private: -L${libdir}  -lldastoolsal ${CXXSTDLDFLAGS} @BOOST_LDFLAGS@ @STDCXXFS_LIBS@
 Cflags: ${CXXSTDFLAGS} -I${includedir} @BOOST_CPPFLAGS@
