diff --git a/ldastoolsal/CMakeLists.txt b/ldastoolsal/CMakeLists.txt
index 58b71d765..482aba3a6 100644
--- a/ldastoolsal/CMakeLists.txt
+++ b/ldastoolsal/CMakeLists.txt
@@ -211,8 +211,20 @@ cm_define(
   VARIABLE LDAS_WORDS_BIGENDIAN
   VALUE ${LDAS_WORDS_BIGENDIAN}
   DESCRIPTION "Set to 1 if system is bigendian" )
-cx_struct_procfs_psinfo( )
-cx_struct_procfs_stat( )
+cm_define(
+  VARIABLE HAVE_PROCFS_PSINFO
+  VALUE 0
+  DESCRIPTION "Defined if /proc/<pid>/psinfo exists"
+)
+cm_define(
+  VARIABLE HAVE_PROCFS_STAT
+  DESCRIPTION "Defined if /proc/<pid>/stat exists"
+)
+if(APPLE)
+  set(HAVE_PROCFS_STAT 0)
+else(APPLE)
+  set(HAVE_PROCFS_STAT 1)
+endif()
 
 #dnl =====================================================================
 #dnl  Check for individual libraries
